cmd/kubeadm/app/cmd/init.go

```diff
@@ -177,11 +182,12 @@ func NewInit(cfgPath string, cfg *kubeadmapi.MasterConfiguration, skipPreFlight
 	// Try to start the kubelet service in case it's inactive
 	preflight.TryStartKubelet()
 
-	return &Init{cfg: cfg}, nil
+	return &Init{cfg: cfg, skipTokenPrint: skipTokenPrint}, nil
```

Comment:
Idea for follow-up: I think we should kill NewInit() and move everything to Init.Run()

That way we wouldn't have to pass everything as args to this function
But can be handled later